lib/java/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
This control file is autogenerated by the Gradle Wrapper integration. It is expected to exist and is updated as Gradle Wrapper is refreshed to a newer version.